install_packages.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+# Script for installing packages and removing apt cache
+
+set -e
+set -u
+export DEBIAN_FRONTEND=noninteractive
+n=0
+max=2
+until [ $n -gt $max ]; do
+    set +e
```

Comment:
Maybe others can comment on whether it is a good practice to switch this off/on.

Reply:
This script was taken from minideb, it seems to be switched on and off so it can handle its own errors and do some retrying.